src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/equalshashcode/Example1.java

```diff
@@ -0,0 +1,58 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="EqualsHashCode"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.equalshashcode;
+// xdoc section -- start
+class Example1 {
+  public int hashCode() { // violation, no valid 'equals'
+    return 0;
+  }
+  public boolean equals(String o) {
+    return false;
+  }
+}
+class ExampleNoHashCode {
+  public boolean equals(Object o) { // violation, no 'hashCode'
+    return false;
+  }
+  public boolean equals(String o) {
+    return false;
+  }
+}
+class ExampleBothMethods1 {
+  public int hashCode() {
+    return 0;
+  }
+  public boolean equals(Object o) { // ok, both methods exist
+    return false;
+  }
+}
+class ExampleBothMethods2 {
+  public int hashCode() {
+    return 0;
+  }
+  public boolean equals(java.lang.Object o) { // ok, both methods exist
+    return false;
+  }
+}
```

Comment:
I'm not sure if this is recommended. The equals(String ..) method was there to show the diff between classes ExampleBothMethods1 and ExampleBothMethods2. 

Please do me a favor....
Run checkstyle latest jar on a test file with the code of this example, but replace
```
  public int hashCode() {
    return 0;
  }
```
with
```
public int hashCode() { return 0; }

```
for all such occurences. I'm almost sure there will be errors but let's try ;)
If you don't know how to run audits, instead you can just checkout to a new branch, update file as mentioned and run mvn validation also